app/lib/differential_expression_service.rb

```diff
@@ -183,6 +185,15 @@ def self.validate_annotation(cluster_file, study, annotation_name, annotation_sc
     cluster = study.cluster_groups.by_name(cluster_file.name)
     raise ArgumentError, "cannot find cluster for #{cluster_file.name}" if cluster.nil?
 
+    if DifferentialExpressionResult.where(study: study,
+                                          cluster: cluster,
+                                          annotation_name: annotation_name,
+                                          annotation_scope: annotation_scope).exists?
+      raise ArgumentError,
+            "#{annotation_name} already exists for #{study.accession}:#{cluster_file.name}, " \
+            'please delete the existing results before retrying'
```

Comment:
Non-blocker: Do these get logged to either Sentry or Mixpanel?  It's unessential now, but seems worthwhile before integrating DE into the upload process.

Reply:
No - none of these get logged to Sentry because they're not indicative of something being broken.  We're just throwing an `ArgumentError` if the user specifies a parameter that would result in an invalid DE job.

Reply:
Gotcha, thanks for clarifying.  If the user error could ever be triggered by non-SCP-staff, I think it'd be worth eventually logging to at least Mixpanel.  But my understanding now is that these errors would never be encountered by an end user, and thus aren't valuable to durably log.